vlei-verifier-workflows/test/run-workflow.test.ts

```diff
@@ -25,10 +29,417 @@ test.only("workflow", async function run() {
   const configFileName = env.configuration;
   let dirPath = "../src/config/";
   const configFilePath = path.join(__dirname, dirPath) + configFileName;
+  console.log("file path: ", configFilePath)
   const configJson = await getConfig(configFilePath);
+
+  const sampleConfig = {
```

Comment:
@lynasberaich great job. I like this explicit config and the test cases. Since this will only be a utility for now, can you pull the sampleConfig and test cases for getIdentifierData into a separate test file. run-workflow.test.ts will be used for workflow testing, and probably will also get refactored into separate workflow related tests. But the nice thing that we've decided for getIdentifierData is that it will be a utility and so it can have its own simple test file to make sure it can process different configs and give proper output/errors.

Reply:
awesome job @lynasberaich 
<img width="1271" alt="image" src="https://github.com/user-attachments/assets/1b5b361e-bb12-484a-ba9c-d9ee4d63951c" />